fsr/fsr.ino

```diff
@@ -1,10 +1,14 @@
+#include "FastADC.h"
```

Comment:
Remove this library and do all of the stuff within setup. Specifically change the function call in setup to be the following:

```cpp
#if defined(_SFR_BYTE) && defined(_BV) && defined(ADCSRA)

#define ClearBit(sfr, bit) (_SFR_BYTE(sfr) &= ~_BV(bit))
#define SetBit(sfr, bit) (_SFR_BYTE(sfr) |= _BV(bit))

// Set the ADC prescaler to 16 for boards that support it,
// which is a good balance between speed and accuracy.
// More information can be found here: http://www.gammon.com.au/adc
SetBit(ADCSRA, ADPS2);
ClearBit(ADCSRA, ADPS1);
ClearBit(ADCSRA, ADPS0);

#undef ClearBit
#undef SetBit

#endif
```

Reply:
Thoughts on wrapping that second part in a function such as SetFastAnalog()?